lxd/patches.go

```diff
@@ -81,6 +81,7 @@ var patches = []patch{
 	{name: "storage_zfs_unset_invalid_block_settings", stage: patchPostDaemonStorage, run: patchStorageZfsUnsetInvalidBlockSettings},
 	{name: "storage_zfs_unset_invalid_block_settings_v2", stage: patchPostDaemonStorage, run: patchStorageZfsUnsetInvalidBlockSettingsV2},
 	{name: "storage_unset_invalid_block_settings", stage: patchPostDaemonStorage, run: patchStorageUnsetInvalidBlockSettings},
+	{name: "remove_candid_rbac_config_keys", stage: patchPreDaemonStorage, run: patchRemoveCandidRBACConfigKeys},
```

Comment:
other patches are named "<thing>_<action>" e.g. `warnings_remove_empty_node` so I suggest we stick with that convention here and call this `candid_rbac_remove_config_keys`.